doc/source/development/extending.rst

```diff
@@ -280,7 +280,7 @@ Property Attributes          ``Series``              ``DataFrame``
 ===========================  ======================= =============
 ``_constructor``             ``Series``              ``DataFrame``
 ``_constructor_sliced``      ``NotImplementedError`` ``Series``
-``_constructor_expanddim``   ``DataFrame``           ``Panel``
+``_constructor_expanddim``   ``DataFrame``           ``NotImplementedError``
```

Comment:
This is not true _yet_, but I would think this would raise a `NotImplementedError` once panel is removed?

Reply:
Makes sense to me. FYI there's a call above this for `to_panel` I imagine we want to remove

Reply:
Nice catch. Removed.